.github/workflows/linting.yml

```
@@ -342,6 +358,26 @@ jobs:
           FILES: ${{ needs.changed_files.outputs.js_excluded_files }}
         run: pnpm run lint-changed --git-base=$SHA $(jq -rn --argjson files "$FILES" '$files[]')
 
+  ### Runs lint-style on all CSS/SCSS files in the monorepo except those ignored in .stylelintignore.
+  # Local equivalent: `pnpm run lint-style .`
+  lint_style:
+    name: Stylelint
+    runs-on: ubuntu-latest
+    needs: changed_files
+    if: needs.changed_files.outputs.css == 'true' || needs.changed_files.outputs.misc_css == 'true'
+    timeout-minutes: 5  # 2025-04-03: Takes about a minute, so give a little wiggle room.
+
+    steps:
+      - uses: actions/checkout@v4
+
+      - name: Setup tools
+        uses: ./.github/actions/tool-setup
+
+      - name: Monorepo pnpm install
+        run: pnpm install
+      - name: Run lint-style
+        run: pnpm lint-style . && echo 'Everything checks out!'
```

I see in [the previous run](https://github.com/Automattic/jetpack/actions/runs/14248859421/job/39936561808) it didn't find annotations for the failures, just "Process completed with exit code 2."

Considering eslint has very similar output, we may be able to fix it by doing like this
```suggestion
      - name: Run lint-style
        env:
          NO_COLORS: 1
        run: pnpm lint-style . && echo 'Everything checks out!'
```

---

Looks like that won't actually work, the formatting is still different. OTOH, this should
```suggestion
      - name: Run lint-style
        run: pnpm lint-style --format=compact . && echo 'Everything checks out!'
```

---

Hmm... Well, maybe not yet, until setup-node gets a release to include [actions/setup-node#98](https://href.li/?https://github.com/actions/setup-node/pull/98). Sigh.